recipes/opencv/4.x/conanfile.py

```diff
@@ -223,6 +227,7 @@ def _configure_cmake(self):
         self._cmake.definitions["WITH_OPENJPEG"] = self.options.with_jpeg2000 == "openjpeg"
         self._cmake.definitions["WITH_OPENEXR"] = self.options.with_openexr
         self._cmake.definitions["WITH_EIGEN"] = self.options.with_eigen
+        self._cmake.definitions["HAVE_QUIRC"] = self.options.with_quirc  # force usage of quirc reqirement
```

Comment:
It doesn't remove vendored quirc: https://github.com/opencv/opencv/blob/8ac304517e0ec2f9ad695b05b94b1ad1d1e82c1c/CMakeLists.txt#L779